root/edit/details/AddSeries.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {addColonText} from '../../static/scripts/common/i18n/addColon';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+
+type AddSeriesEditProps = {
+  ...EditT,
+  +display_data: {
+    +comment: string,
+    +name: string,
+    +ordering_type: SeriesOrderingTypeT,
+    +series: SeriesT,
+    +type: SeriesTypeT,
+  },
+};
+
+type Props = {
+  edit: AddSeriesEditProps,
+};
+
+const AddSeries = ({edit}: Props) => {
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Series'))}</th>
+            <td><EntityLink entity={edit.display_data.series} /></td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details add-series">
```

Comment:
I know this is in the original code, but I can't figure out why we have these two as separate tables...

Reply:
On second thought, you can leave this be for now and we can fix them all in one commit later.

Reply:
Okay !